CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-kubernetes-gateway/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 0.6.0
+
+*August 31, 2023*
+
+This release adds a Helm chart, dynamic control plane logging, Prometheus metrics, and in-depth guides for various use cases.
+
+FEATURES:
+
+- Helm chart. [PR-840](https://github.com/nginxinc/nginx-kubernetes-gateway/pull/840)
+- Use custom nginx container. [PR-934](https://github.com/nginxinc/nginx-kubernetes-gateway/pull/911)
+- Support dynamic control plane configuration. [PR-943](https://github.com/nginxinc/nginx-kubernetes-gateway/pull/943)
```

Comment:
To a user, isn't this really more about controllable log levels during runtime?

Reply:
Yep, I'll be more specific